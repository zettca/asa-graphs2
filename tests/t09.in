300 1000
174
151 33 -20
66 99 1
17 262 19
35 137 17
35 163 18
56 11 29
22 85 -21
298 171 1
105 163 2
41 154 30
26 171 26
129 244 -27
121 25 15
135 295 -22
151 15 4
82 293 21
59 53 15
100 176 30
275 228 26
212 226 7
155 18 28
299 151 26
135 269 -6
20 112 -17
137 258 15
279 290 21
165 271 11
160 158 13
78 113 25
298 85 -23
207 137 25
135 100 -5
73 189 20
105 131 -6
134 259 -12
103 126 30
65 101 18
183 87 22
15 39 -9
255 84 16
70 106 21
94 192 8
78 256 -6
32 196 14
130 124 16
134 276 30
167 83 27
157 137 -21
212 112 23
149 249 25
214 183 -5
133 185 -13
131 69 -19
240 198 29
94 298 -21
211 8 -21
119 116 17
67 265 -28
219 199 4
199 188 -28
267 140 7
219 6 -5
205 147 22
267 294 27
146 106 30
290 165 23
190 221 12
211 226 7
175 126 0
86 31 -7
108 89 23
58 188 -12
190 284 16
38 209 20
266 39 27
174 172 1
237 23 -23
254 220 16
257 155 27
254 77 13
227 33 -14
257 69 -11
282 173 25
1 53 5
42 70 -20
77 89 24
68 119 5
104 18 9
78 220 7
245 62 4
66 229 3
170 17 28
90 297 9
291 113 18
163 45 -8
59 248 -8
17 159 -17
77 115 3
237 283 13
81 136 3
120 203 -19
1 95 15
158 205 -5
236 103 -30
187 25 15
230 26 26
91 193 -21
283 207 26
191 124 28
45 118 5
209 239 8
172 112 15
180 124 -29
152 104 -13
67 6 -4
72 240 18
79 224 -27
85 245 -14
87 254 8
103 284 23
110 71 9
177 97 24
223 70 24
172 238 29
191 213 -11
203 252 3
238 73 -1
3 145 6
257 284 -11
216 145 -5
62 243 30
155 4 -29
189 137 -15
71 136 25
113 191 14
86 45 26
44 146 24
77 245 8
216 243 1
39 100 22
208 172 -3
171 296 27
227 179 -2
132 117 -4
277 134 21
158 221 28
74 5 -13
279 225 24
152 255 25
160 132 28
246 215 8
254 197 11
118 50 -3
230 239 11
288 45 15
150 96 28
175 3 17
284 206 -16
10 163 -5
73 141 -18
116 118 25
20 250 -3
152 64 2
225 139 -20
78 89 4
130 101 29
198 22 -22
204 294 -4
150 72 0
182 97 -30
51 19 -19
59 43 -17
214 215 -2
135 251 22
32 174 20
202 245 14
56 221 1
51 64 -23
166 220 12
204 49 -29
204 141 -30
99 226 10
109 53 18
197 114 18
230 43 -7
149 52 -29
294 39 17
30 295 20
87 96 11
182 115 -11
254 226 14
135 77 -6
150 3 25
70 131 28
44 107 0
286 179 -2
56 90 -16
262 84 -6
239 192 9
222 132 29
157 190 10
147 85 -20
82 190 30
237 263 25
81 40 10
278 40 -22
265 28 -15
219 33 10
176 194 26
222 68 16
55 125 19
190 146 4
265 55 -12
114 175 26
182 210 -10
228 52 -22
143 245 -24
24 169 -2
2 252 -25
90 49 14
129 241 3
222 230 9
251 226 22
56 65 29
174 281 -6
135 220 -14
181 30 18
151 235 26
193 9 24
42 126 9
68 125 26
220 29 0
123 172 15
204 63 10
22 189 20
96 197 -8
127 168 8
210 77 -1
211 58 26
54 156 7
213 242 29
242 144 11
263 197 5
263 60 9
186 31 -23
260 95 4
116 192 0
196 52 -18
19 245 10
143 39 18
197 116 -1
209 126 17
294 148 -19
33 41 20
66 202 22
49 189 -5
220 207 -2
92 155 -3
231 198 5
161 273 25
216 14 23
102 82 -7
286 65 16
77 248 22
35 212 26
289 77 -11
52 74 30
9 24 15
180 155 12
267 262 -1
94 131 17
113 261 17
266 231 -13
194 140 13
206 149 27
123 100 -2
26 55 14
2 93 -1
201 85 15
224 285 2
161 227 20
60 97 -13
221 247 29
213 293 18
11 280 30
58 4 27
165 253 -14
184 182 26
43 128 -7
51 241 -28
242 209 21
51 212 -27
11 300 12
138 230 9
142 45 6
290 246 24
88 41 21
175 194 -8
199 280 0
214 289 27
266 97 -18
183 100 24
180 240 10
152 124 -11
153 154 -12
164 272 -12
22 272 -6
142 100 -20
70 139 20
243 57 19
79 193 -20
97 21 22
189 118 -6
140 71 24
277 95 17
212 7 7
8 120 9
58 233 -7
51 243 7
129 221 23
253 87 21
300 259 -23
261 121 19
218 129 9
145 129 7
32 179 13
213 212 6
228 222 6
286 163 -26
297 5 19
235 157 11
229 158 2
299 282 -3
275 81 30
285 206 20
262 263 -2
231 54 21
9 53 -18
132 269 20
276 22 -10
26 271 -12
180 250 17
150 171 15
230 253 19
285 262 1
147 121 28
279 198 13
68 151 18
247 261 -28
278 55 21
279 112 -16
154 193 29
54 62 -16
255 182 8
172 60 27
208 160 26
296 167 20
126 19 15
225 196 26
109 156 -16
126 80 12
53 284 14
152 62 4
123 142 26
256 78 10
288 167 9
245 57 -5
131 113 -24
127 150 24
231 72 4
278 293 27
240 43 -15
80 172 -22
65 179 0
224 180 -10
118 299 13
99 7 -24
23 184 14
251 185 27
40 222 8
248 253 28
147 229 4
42 175 11
117 215 14
98 210 24
217 5 -21
228 239 21
59 55 -11
159 81 26
275 63 -2
242 13 18
28 240 24
99 69 28
175 223 12
87 97 19
120 286 27
48 128 26
56 187 -9
113 124 21
148 98 25
247 273 -14
81 206 -8
115 37 30
87 294 -17
26 85 -6
153 96 -22
132 292 19
264 241 16
294 285 23
112 284 20
10 261 -3
187 101 23
118 95 24
93 235 -18
194 223 22
161 130 -6
152 33 25
254 179 -1
276 188 -3
154 139 28
290 114 11
3 113 -10
243 151 24
236 18 20
120 201 22
251 84 17
76 159 -20
80 144 29
119 254 27
75 27 -2
26 228 -19
54 73 20
59 191 -3
190 293 28
170 84 -3
82 57 -25
204 271 -18
15 113 26
91 298 30
192 190 30
95 221 7
185 161 27
170 202 20
51 119 -15
291 169 29
290 159 30
9 68 0
93 91 27
151 6 -4
206 176 9
160 190 20
205 187 7
88 160 25
230 1 -1
267 108 -6
69 78 21
86 2 -29
2 151 -7
149 259 4
298 75 7
89 240 26
5 98 20
139 202 -20
279 218 -11
290 36 28
29 171 18
193 215 2
93 52 8
296 143 2
116 41 25
261 230 4
182 33 3
261 181 -2
33 270 17
197 44 11
76 20 17
186 254 11
23 146 -29
251 133 -29
285 281 -22
19 104 22
10 126 20
150 165 -3
145 155 -18
74 277 -15
24 284 -7
122 206 11
284 117 30
223 145 9
136 232 15
85 291 25
268 105 21
143 275 8
214 151 5
285 141 -17
288 227 -26
164 288 11
109 185 22
266 222 -28
219 104 -21
53 17 -19
94 251 -7
254 137 -26
22 144 18
184 287 23
172 160 20
58 80 -12
64 151 19
90 78 29
201 203 0
179 299 -7
94 47 25
11 44 19
284 202 -8
223 65 -11
96 287 6
60 134 -6
138 15 -29
15 195 -17
76 211 29
153 198 23
241 267 11
184 160 3
23 110 -2
138 45 -14
285 161 17
254 249 24
295 64 -2
145 29 13
86 258 6
115 158 15
167 261 9
252 61 12
254 205 -9
10 50 16
177 194 8
128 12 24
257 67 9
47 189 29
154 121 -1
92 233 19
34 88 30
70 102 28
74 31 26
253 202 -25
194 100 -7
219 110 19
171 17 -27
180 138 11
130 30 -13
156 95 -18
51 65 25
278 162 16
138 3 -22
51 169 28
67 278 22
187 65 11
102 155 -3
204 182 -26
251 112 24
172 74 23
99 59 -22
239 236 23
145 47 -19
217 18 9
130 82 24
73 74 23
235 129 26
126 294 5
230 206 16
150 229 22
207 136 21
151 81 16
178 9 28
74 43 2
276 73 15
235 263 2
152 112 9
26 210 -17
108 40 8
250 156 -12
265 17 -3
271 154 17
121 33 14
71 160 1
212 196 18
8 247 -17
147 217 6
108 97 30
200 76 30
200 12 -13
218 148 28
194 102 1
150 18 -23
169 246 19
265 264 10
194 17 19
119 70 27
16 155 -13
84 155 2
281 184 12
266 31 -9
82 223 26
188 285 23
67 240 -28
202 238 25
151 50 -2
284 194 -6
71 290 17
188 260 19
20 222 24
252 107 -11
59 281 0
48 62 -14
208 273 -24
95 282 -25
37 186 -2
110 155 -16
24 147 30
18 161 11
150 233 10
155 57 4
219 207 -23
288 295 -1
250 25 -6
89 246 19
242 33 -13
139 55 9
297 62 3
238 90 3
209 185 0
199 23 -30
299 172 6
238 22 -5
24 156 18
204 66 9
176 73 14
171 184 -29
258 111 29
123 137 28
72 123 8
8 173 30
72 202 -6
247 112 19
231 38 5
132 74 29
45 265 2
230 24 26
286 153 -2
32 210 30
25 176 -5
272 71 13
132 207 -25
190 144 24
86 189 -19
254 218 15
283 173 7
161 41 7
246 71 30
247 110 -10
239 82 -8
289 118 30
59 144 -9
58 69 -9
19 213 15
269 110 7
170 261 -2
219 297 -20
280 155 16
176 277 10
235 87 10
170 134 21
232 272 18
117 223 -1
107 121 30
138 199 -28
165 170 -25
23 122 15
149 68 10
185 125 -17
32 107 24
291 242 -15
137 108 28
78 154 3
44 116 -15
151 221 1
189 182 25
271 114 -26
71 255 5
150 124 2
21 277 -8
248 9 26
7 272 21
23 111 11
285 170 24
98 60 3
195 113 20
96 39 30
166 32 25
104 267 6
156 143 -13
226 276 22
61 89 -17
298 152 -3
253 249 29
128 197 9
206 18 15
138 186 -11
132 294 -20
93 148 -12
182 166 7
196 291 -9
279 33 25
258 122 -15
75 144 -1
8 237 29
171 102 -6
107 209 5
40 197 21
156 19 2
239 238 17
52 214 13
126 140 -15
83 132 -30
229 177 25
293 210 -22
274 190 16
212 19 9
133 42 6
17 213 24
258 75 -2
101 188 -19
29 140 -22
122 281 11
199 123 16
151 128 20
260 74 -7
55 19 14
272 126 24
217 118 23
252 9 23
105 255 -7
21 7 6
84 235 15
156 111 9
136 97 -1
288 13 -22
266 261 25
118 53 12
47 146 8
208 1 3
207 287 -16
249 248 -10
268 62 19
114 264 23
57 25 21
56 142 1
279 118 26
199 103 -28
158 47 13
72 57 -15
113 282 15
72 185 5
109 268 -1
200 256 22
112 265 5
239 108 -7
262 300 -7
102 11 7
240 110 24
247 246 27
102 275 -3
151 132 11
67 254 24
157 44 15
129 184 7
246 21 12
153 179 -6
225 237 25
151 162 -19
118 273 1
225 202 -17
36 148 -2
280 234 29
94 198 -11
59 203 9
169 38 -16
12 226 -19
262 81 30
227 182 20
95 35 22
37 264 0
153 236 27
73 78 16
118 110 -10
213 249 23
265 15 7
112 222 -18
261 28 14
237 271 -27
93 242 -1
180 115 18
45 203 -10
203 168 23
41 173 -2
39 106 12
217 25 -30
34 124 27
109 234 -8
164 287 6
290 75 25
271 292 9
131 87 -23
111 229 -10
291 174 30
21 12 27
54 132 21
8 171 -2
195 172 24
246 253 7
90 238 8
8 154 20
232 142 11
29 3 21
198 232 -28
268 39 17
167 112 16
230 128 -6
42 79 8
47 28 21
72 5 15
278 23 5
21 227 18
286 221 0
163 1 -10
251 206 -17
243 191 17
234 88 -14
61 98 -4
250 5 17
294 130 24
34 220 -30
30 272 10
2 204 28
190 196 1
184 159 11
221 38 -10
173 255 14
247 104 -15
193 5 -28
122 88 4
274 121 7
255 32 -3
279 142 21
62 173 27
196 75 28
20 72 22
271 92 13
45 90 7
139 144 5
60 162 13
238 168 -29
293 27 -11
234 290 -7
269 114 -16
204 61 -4
236 159 16
64 45 29
42 41 -2
229 129 6
185 172 30
179 7 27
152 253 6
201 33 -18
207 226 22
68 50 17
98 24 13
257 34 9
29 17 -8
64 209 -3
247 48 29
127 219 -25
72 102 -8
245 32 20
251 265 -28
208 13 20
204 247 22
278 109 17
251 220 14
233 137 -6
150 22 9
51 114 -24
263 46 -1
79 13 -15
84 4 -6
59 256 -10
10 128 29
186 55 -13
121 149 6
72 60 -13
231 41 -5
75 205 28
157 261 16
88 218 22
133 233 24
53 37 10
106 296 8
191 247 21
208 169 -15
217 173 11
201 177 -19
29 169 -26
283 155 13
164 299 -12
246 281 -17
91 266 30
228 205 13
165 75 23
233 44 -23
13 300 26
235 47 27
127 169 23
198 216 22
158 267 -9
138 137 17
90 108 4
284 241 8
292 240 -2
26 239 30
18 87 28
10 83 -17
23 131 -18
238 48 26
158 255 -8
110 81 10
274 197 11
138 239 18
15 143 -7
219 157 -12
10 1 2
20 70 16
111 140 29
278 141 16
2 53 12
248 142 5
96 16 18
274 58 16
209 178 16
85 33 8
250 85 14
277 262 24
104 29 -1
28 252 -1
76 151 13
153 46 28
83 290 26
79 259 20
17 280 -2
27 17 6
250 257 20
291 173 8
174 132 24
224 284 -18
290 145 7
290 29 24
81 259 -28
34 176 -5
58 245 -4
231 232 27
14 228 -9
273 207 18
159 31 -20
186 243 30
246 175 7
42 62 -21
97 106 30
16 198 22
146 256 24
216 185 18
153 142 18
227 291 13
51 200 -16
268 93 25
69 153 -5
236 299 19
254 75 21
6 246 5
73 14 24
98 193 16
137 35 19
97 48 11
256 60 -7
204 22 -29
40 116 8
160 122 -25
210 97 -22
42 163 29
77 181 -13
112 217 5
125 195 16
