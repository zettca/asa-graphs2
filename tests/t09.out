6
U
33
-41
-7
4
4
U
28
U
13
17
3
24
12
38
-9
7
19
U
-2
10
U
18
-9
33
9
6
22
-1
-46
-5
-28
U
21
39
20
-20
3
8
-8
U
0
13
30
2
-1
-13
33
6
U
-10
10
U
5
U
-30
U
U
3
17
-27
1
17
3
U
U
14
47
38
13
U
0
14
15
U
-3
16
U
28
0
-1
61
-1
-11
U
20
-14
0
19
U
34
U
U
7
20
-24
5
U
-2
21
6
0
23
U
6
-6
0
U
4
45
16
15
-5
0
-2
15
14
19
U
18
-18
U
28
-18
16
U
-7
17
12
66
19
U
-3
U
3
2
U
47
0
-18
24
5
-4
18
7
32
-20
19
U
8
U
42
19
-12
36
25
15
-26
7
18
-11
39
U
34
19
34
-13
-4
9
40
1
-10
0
16
-14
60
15
3
U
-16
12
-2
4
-1
18
17
2
20
27
20
-2
21
5
-5
9
2
55
U
U
U
-9
20
U
10
-8
-6
U
-1
-2
U
15
9
3
1
77
21
8
U
23
9
-2
14
U
U
-2
16
14
35
7
U
18
53
18
14
30
U
16
7
26
19
-15
3
-10
-25
9
38
19
32
U
U
5
16
28
4
10
U
17
-28
21
7
5
3
18
21
U
6
U
39
-11
21
4
15
U
3
20
-10
U
U
-11
-8
-18
U
11
22
U
-22
U
30
11
0
30
20
-1
19
14
28
U
-4
-2
